### DISCUSSION TITLE
TailwindCSS not work with Rollup and publish npm

### DISCUSSION THREAD

**Discussion by MikaelAugustoDev** [tailwindlabs/tailwindcss#12038] [Help]
I am testing publish npms, but tailwindCSS does not load when I import the package that contains tailwindcss and components minify with rollup, however, when I use some styling on the page that is the same as the styling of my published component, the tailwind styling works, help me

**Comment by wongjn** [comment]
It sounds like the CSS generated by Tailwind in the package is not included in the webpage of the consumer project. Consider providing some remotely-hosted git repositories that reproduce the unexpected behavior.
